resources/js/admin.js

```diff
@@ -0,0 +1,4 @@
+import "../css/plugin.css";
```

Comment:
@Perumal605 , we should add an admin.css file, rather than plugin.css. We won't need the frontend styles on the backend.

Reply:
@incraigulous , I have added admin.css file and modified admin.js file.

Reply:
@incraigulous I have committed changes to install DT web components with this commit.